src/test/kotlin/org/openbroker/se/privateunsecuredloan/TestObjectsJson.kt

```diff
@@ -111,7 +111,8 @@ object TestObjectsJson {
 					"monthlyPayment": 22,
 					"shouldRefinance": true,
 					"existingLoanType": "CAR_LOAN",
-					"responsibility": "SHARED"
+					"responsibility": "SHARED",
+                    "lenderName": "banksy"
```

Comment:
It seems like the indent is wrong :thinking:

Reply:
All tough I have a strong suspicion that it might be because there are some tabs instead of spaces in there, in which case it is almost certainly my fault...

Reply:
You are most likely correct about the tabs. "Fixed" it now by copying the spaces from the row above.